darts/tests/models/forecasting/test_backtesting.py

```diff
@@ -194,6 +223,8 @@ def test_backtest_forecasting(self):
             self.assertEqual(pred.width, 2)
             self.assertEqual(pred.end_time(), linear_series.end_time())
 
+            # TODO: adjusted window size hard to test with tcn model
```

Comment:
Is this comment still neeed?

Reply:
I wanted to make sure we still add a unit test of the sort you suggested, that accuracy is still OK with one of the not so simple models.